auth/email_action_links.js

```diff
@@ -71,14 +71,14 @@ admin
 
 // [START sign_in_with_email_link]
 // Admin SDK API to generate the sign in with email link.
-const usremail = 'user@example.com';
+const useremail = 'user@example.com';
```

Comment:
`useremail` already exists as a `const` above, so we'll have to use a new variable name.

Reply:
@samtstern Then I'd suggest to be consistent with the other examples and reduce it to a single `const` at the top of the file?

Reply:
@peili yes I think you're right that one function per snippet would be the better approach, just like we do over here:
https://github.com/firebase/snippets-web/blob/master/functions/callable.js#L17